Fetch/ResponseError.swift

```diff
@@ -14,3 +14,12 @@ import Foundation
 public enum ResponseError: Error {
     case statusCode(Int)
 }
+
+public extension ResponseError {
```

Comment:
This might not work, due to the way `Error` works internally. The suggested way to achieve this is:

```
public extension ResponseError: LocalizedError {
     public var errorDescription: String? {
        switch self {
         case .statusCode(let code):
             return "Status code error: \(code)"
         }
     }
 }
```

Reply:
Ah ok, ta.